packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -320,19 +353,25 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
       templateImageDirName: await app.templateLaunchImageDirNameForImages,
       projectImageDirName: app.projectLaunchImageDirName);
 
-    if (usesTemplate) {
-      messageBuffer.writeln('\nWarning: Launch image is set to the default placeholder. Replace with unique launch images.');
-    }
+
+    final ValidationMessage? templateValidationMessage = _createValidationMessage(
+        isValid: !usesTemplate,
+        messageIfValid: 'Launch image is not set to the default placeholder image.',
+        messageIfInvalid: 'Launch image is set to the default placeholder icon. Replace with unique launch image.');
```

Comment:
A trailing comma will allow this whitespace to be fixed.

Reply:
wow